bat.fish

```diff
@@ -0,0 +1,59 @@
+# Fish Shell Completions (WIP)
+
+# Place or symlink to $XDG_CONFIG_HOME/fish/completions/bat.fish ($XDG_CONFIG_HOME is usually set to ~/.config)
+complete -c bat -s l -l language -d "Set the language for syntax highlighting"
+
+complete -c bat -l list-languages -d "Display list of supported languages for syntax highlighting"
+
+complete -c bat -s m -l map-syntax -d "<from:to> Map a file extension or file name to an existing syntax"
+
+# TODO: add completion for theme list
```

Comment:
`bat --list-themes` does produce a newline-separated list of theme names if it is run in non-interactive mode, i.e. if the output is piped somewhere.